package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@moxy/next-runtime-env",
+  "version": "0.0.0",
```

Comment:
@satazor should we start the version at `1.0.0` and publish that or should we have a commit for the version bump that is used to publish to npm?

Reply:
Let's start with v0.0.0 and use `npm run release -- --release-as 1.0.0` to bump, which in turn uses `standard-version`

Reply:
Sure, added command for `release`.